types/katex.d.ts

```diff
@@ -20,6 +20,43 @@ export type TrustContext =
     | { command: "\\htmlStyle", style: string }
     | { command: "\\htmlData", attributes: Record<string, string> }
 
+
+type KatexCatcodes = Record<string, number>;
```

Comment:
I don't think you need to prefix all the types and interfaces with `Katex`. And probably they should all be `export`ed too.